walk-in-interview/src/main/webapp/new-job/new-job.js

```diff
@@ -0,0 +1,293 @@
+/**
+ * This file is specific to new-job.html. It renders the fields on the
+ * page dynamically, and it also makes the POST request when the form
+ * is submitted.
+ */
+
+import {AppStrings} from './strings.en.js';
+
+const NEW_JOB_STRINGS = AppStrings['new-job'];
```

Comment:
FYI you only need NEW_JOB_STRINGS there, so maybe just call it STRINGS? and if you need non 'new-job' strings (ex: you have common ones like product name and such) then you can have a COMMON_STRINGS ?